### PR TITLE
Every even added

### PR DESCRIPTION
**Pull request by Regggis**
Every even added. But still thinking about some optimazation :D

### CODE REVIEW

**Comment by ziobron** [comment]
✅ CI Passed - Task: add-even
🏅 Granted 5 XP for @Regggis

**Comment by ziobron** [comment]
⏱ 3 XP extra for @Regggis for on-time delivery

File: homework/add-even/addEven.hpp [hunk 1]
```diff
@@ -2,7 +2,15 @@
...
-    return -1;
+    
+    int evenAdded = 0;
+    for(auto value : numbers)
+    {
+        if(value%2 == 0)
+        {
+            evenAdded +=value;
+        }
+        else{}
```
Comment (nauka-programowania-MA) [inline homework/add-even/addEven.hpp:13]: remove

File: homework/add-even/addEven.hpp [hunk 2]
```diff
@@ -2,7 +2,15 @@
...
 
 int addEven(const std::vector<int>& numbers) {
-    // TODO: Your implementation goes here
-    // Below return is only to make this function compile now
-    return -1;
+    
+    int evenAdded = 0;
+    for(auto value : numbers)
+    {
+        if(value%2 == 0)
```
Comment (nauka-programowania-MA) [inline homework/add-even/addEven.hpp:9]: Add space between operations like '+' '-' '!=' '==' '%' etc...

**Review by nauka-programowania-MA — APPROVED** [review]
Optimization will be `std::accumulate` :) We will learn this later

File: homework/add-even/addEven.hpp (hunk 1 above)
Comment (Regggis) [reply homework/add-even/addEven.hpp:13]: You mean "else" is not necessary, yes?

Comment (nauka-programowania-MA) [reply homework/add-even/addEven.hpp:13]: Yes, this is redundant :)

**Pull request closed without merging**